WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -748,6 +760,7 @@ public boolean deleteBlog(Context ctx, int id) {
         int rowsAffected = db.delete(BLOGS_TABLE, "id=?", new String[]{Integer.toString(id)});
         deleteQuickPressShortcutsForLocalTableBlogId(ctx, id);
         deleteAllPostsForLocalTableBlogId(id);
+        PeopleTable.deletePeopleForBlog(id);
```

Comment:
I think it would be a good idea to rename this to `deletePeopleForLocalBlogId` to make it clear it's a **local** blog ID.